code/modules/core_implant/cruciform/machinery/eotp.dm

```diff
@@ -0,0 +1,197 @@
+var/global/obj/machinery/power/eotp/eotp
+
+#define ARMAMENTS "Armaments"
+#define ALERT "Antag Alert"
+#define INSPIRATION "Inspiration"
+#define ODDITY "Oddity"
+#define STAT_BUFF "Stat Buff"
+#define MATERIAL_REWARD "Materials"
+
+
+/obj/machinery/power/eotp
+	name = "Eye of the Protector"
+	desc = "He observe, he protects."
+	icon = 'icons/obj/eotp.dmi'
+	icon_state = "Eye_of_the_Protector"
+
+	density = TRUE
+	anchored = TRUE
+	layer = 5
+
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 30
+	active_power_usage = 2500
+
+	var/list/rewards = list(ARMAMENTS = 60, ALERT = 30, INSPIRATION = 70, ODDITY = 40, STAT_BUFF = 50, MATERIAL_REWARD = 90)
+
+	var/list/materials = list(/obj/item/stack/material/platinum/random = 40,
+									/obj/item/stack/material/steel/random = 70,
+									/obj/item/stack/material/gold/random = 15,
+									/obj/item/stack/material/uranium/random = 10,
+									/obj/item/stack/material/plasma/random = 20,
+									/obj/item/stack/material/plastic/random = 50,
+									/obj/item/stack/material/plasteel/random = 30,
+									/obj/item/stack/material/wood/random = 60,
+									/obj/item/stack/material/glass/random = 60,
+									/obj/item/stack/material/biomatter/random = 50,
+									/obj/item/stack/material/silver/random = 20)
```

Comment:
Common materials shouldn't be obtainable here. No wood, glass, steel, biomatter (really?), or plastic. None of that.

Make the Eye give more mats of the remaining stacks. At least 30 for the rarest of materials (plasma) and a full stack of 120 for the most common of the rare materials (plasteel).

This should be a massive, massive reward for NT, since they need to work together to make it happen. Why would they work together for a grand reward of 50 biomatter?

Reply:
Done.